improver_tests/acceptance/SHA256SUMS

```diff
@@ -345,6 +345,9 @@ cced29f107253f8a377d562e60c5a41fbf2104ec4bb07dc450aa4912a2680acb  ./map-to-timez
 f7eec028502a0c233149362f0f839f5856d31c7ff1469b35b7a9ebb56f85a2a8  ./map-to-timezones/uk/input_20201203T0200Z.nc
 8b8e7cc56fbad621683fdc4d2c198e62c7eeb3d462ffdccf7eda2c8b6621ac22  ./map-to-timezones/uk/kgo.nc
 0829e33a7e1508bc646f38744369c977f2c89e34645d5f75a8afffae1f446f9d  ./map-to-timezones/uk/timezone_mask.nc
+a8b1b554438308b4e48b8b5647c7a660c32755108b17088d5eb8249f90e923d0  ./max-in-time-window/input_PT0029H00M.nc
+ecc913d410d71ac52e8dd0f745c44a88ec0f9773a1f7c62859195bb5b21e7c2a  ./max-in-time-window/input_PT0032H00M.nc
+56bb0cd04573e9cd546615037f36c89f4977c4a88c444611e445de0281a7b8a9  ./max-in-time-window/kgo.nc
```

Comment:
This KGO raises this error from the `MetadataInterpreter`

```
ValueError: Cell method maximum: time on probability data should have comment 'of lwe_thickness_of_precipitation_amount'
Cell method sum: time (of lwe_thickness_of_precipitation_amount over 3.0 hours within time window) on probability data should have comment 'of lwe_thickness_of_precipitation_amount'
```

Please can you supply an update to the interpreter to accept this metadata change.